### PR TITLE
Added entities and mappers

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

File: src/test/java/amazon/review/ProductMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class ProductMapperTest {
+    private static final ProductMapper productMapper = new ProductMapper();
```
Comment (MaliukDaria) [inline src/test/java/amazon/review/ProductMapperTest.java:16]: если это константа, то ее имя должно быть большими буквами

File: src/test/java/amazon/review/ProductMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        reviewDtoWithoutProduct.setScore(1L);
+        reviewDtoWithoutProduct.setTime(LocalDateTime.ofInstant(Instant.ofEpochSecond(
+                1346976000L), ZoneId.systemDefault()));
+        reviewDtoWithoutProduct.setSummary("Not as Advertised");
+        reviewDtoWithoutProduct.setText("Product arrived labeled as Jumbo Salted Peanuts...");
+    }
+
+    @Test
+    public void mapOk() {
+        Product expected = new Product("B001E4KFG0");
```
Comment (MaliukDaria) [inline src/test/java/amazon/review/ProductMapperTest.java]: ид продукта можно вынести в константу

File: src/test/java/amazon/review/ProductMapperTest.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/test/java/amazon/review/ProductMapperTest.java:16]: it's not a constant, https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

File: src/main/java/amazon/review/model/ReviewDto.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package amazon.review.model;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+
+@Data
+public class ReviewDto {
```
Comment (RomanDubovskyi) [inline src/main/java/amazon/review/model/ReviewDto.java]:
```suggestion
public class ReviewCsvDto {
```
usually, you have different dtos and it's good to be specific

File: src/main/java/amazon/review/model/mapper/UserMapper.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import amazon.review.model.ReviewDto;
+import amazon.review.model.Role;
+import amazon.review.model.User;
+import java.util.Set;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+    private static final String PASSWORD = "1111";
```
Comment (RomanDubovskyi) [inline src/main/java/amazon/review/model/mapper/UserMapper.java]:
```suggestion
    private static final String DEFAULT_PASSWORD = "1111";
```

File: src/test/java/amazon/review/ProductMapperTest.java (hunk 2 above)
Comment (RomanDubovskyi) [reply src/test/java/amazon/review/ProductMapperTest.java]: \+
or you can declare it as a local variable but without the name, it's not clear why exactly this value is used B001E4KFG0

**Review by RomanDubovskyi — COMMENTED** [review]
good work, added minor comments

File: src/main/java/amazon/review/model/ReviewDto.java (hunk 3 above)
Comment (julia-shvabovich) [reply src/main/java/amazon/review/model/ReviewDto.java]: done

File: src/main/java/amazon/review/model/mapper/UserMapper.java (hunk 4 above)
Comment (julia-shvabovich) [reply src/main/java/amazon/review/model/mapper/UserMapper.java]: done

File: src/test/java/amazon/review/ProductMapperTest.java (hunk 2 above)
Comment (julia-shvabovich) [reply src/test/java/amazon/review/ProductMapperTest.java]: done

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**